Comp2023/src/main/java/frc/robot/commands/WristMoveToAngle.java

```diff
@@ -0,0 +1,56 @@
+
+// ROBOTBUILDER TYPE: Command.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.ARMConsts.WristAngle;
+import frc.robot.subsystems.Arm;
+
+/**
+ *
+ */
+public class WristMoveToAngle extends CommandBase
+{
+  private final Arm  m_wrist;
+  private WristAngle m_Angle;

```

Comment:
m_Angle should be m_angle to match our style.

Reply:
fixed